framework/DataObjects/XDataSet.py

```diff
@@ -1054,6 +1056,30 @@ def _convertToXrDataset(self):
       self._clearAlignment()
     return self._data
 
+  def _checkRealizationFormat(self,rlz):
```

Comment:
while the other methods in this class are alphabetical, this one is out of place.  Not 100% required to change, but it would improve future development of the code.

Reply:
moved